test/unit/matrix/test_panel.cpp

```diff
@@ -378,17 +379,31 @@ void testShrink(const config_t& cfg, const comm::CommunicatorGrid& comm_grid) {
       const LocalTileIndex idx(coord, k);
       dlaf::internal::transformLiftDetach(dlaf::internal::Policy<Backend::MC>(), setTile,
                                           panel.readwrite_sender(idx), counter++);
-      const auto& tile = panel.read(idx).get();
+
+      auto panel_tile_f = panel.read(idx);
+      const auto& panel_tile = panel_tile_f.get();
+      auto matrix_tile_f = matrix.read(idx);
+      const auto& matrix_tile = matrix_tile_f.get();
+
+      // TODO: Comment about why the gets are up there above the SCOPED_TRACE?
+      // This was again a case of gtest expecting SCOPED_TRACE to be destroyed
+      // on the same thread it was constructed.
       SCOPED_TRACE(message);
-      EXPECT_EQ(tile.size(), matrix.read(idx).get().size());
+      EXPECT_EQ(panel_tile.size(), matrix_tile.size());
     }
 
     counter = 0;
     for (const auto& idx : panel.iteratorLocal()) {
+      auto panel_tile_f = panel.read(idx);
+      const auto& panel_tile = panel_tile_f.get();
+      auto matrix_tile_f = matrix.read(idx);
+      const auto& matrix_tile = matrix_tile_f.get();
+
+      // TODO: Was it intentional to do two panel.reads, on for CHECK_TILE_EQ
+      // and one for EXPECT_EQ?
```

Comment:
I don't see a reason to be intentional. In both it behaves the same.

Reply:
I've removed the comment.